nemo/collections/asr/models/ctc_models.py

```diff
@@ -241,7 +241,8 @@ def transcribe(
                     if logprobs:
                         # dump log probs per file
                         for idx in range(logits.shape[0]):
-                            hypotheses.append(logits[idx][: logits_len[idx]])
+                            lg = logits[idx][: logits_len[idx]]
```

Comment:
can you do the same for `ctc_bpe_models.py`, and the 2 rnnt models?

Reply:
ctc_bpe_models.py uses the same transcribe method.

Reply:
RNNT?

Reply:
RNNT models do not have the feature of logprobs. So I added that feature. Please check them to make sure I have not broke anything.

Reply:
No youre right, I forgot RNNT logprobs are not calculated at all during inference. This is good enough.